CaloMC/src/SConscript

```diff
@@ -73,7 +74,7 @@ helper.make_plugins( [ mainlib,
                        'HepPDT',
                        'boost_filesystem',
                        'boost_system',
-                       rootlibs
+                       rootlibs  #needed for reco hit fitting
                      ],
```

Comment:
The comment about "reco" triggered this.   Something to think about for later: usually root fits are slow; they are great for development.  If you are doing simple fits, and if time an issue, it might make sense to have a project down the road to replace the root fits with something faster.

Reply:
My answer has two parts:
1) I use TMinuit directly, which isn't the fastest algorithm on the market but its faster than the root wrapper 
2) The auto-regressive fit is only performed once at the beginning of the run, so the performance penalty is limited

But if we want to switch to a faster minimizer down the road, let me know

Reply:
Thanks.   Since it's once at begin run, it is a super low priority to bother with switching to a faster minimizer.